<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
    background-color: rgb(255, 170, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="allPlayers">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Twój zespół:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Kolejka:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: rgb(255, 0, 0);
        border-radius: 20px;
        border: none;
        color: black;
        padding: 10px 24px;
        font-size: 16px;
    }
QPushButton:hover {
    background-color: rgb(255, 127, 127);  /* Nowy kolor tła przy najechaniu myszką */
}

QPushButton:pressed {
    background-color: rgb(191, 0, 0);  /* Nowy kolor tła po wciśnięciu przycisku */
}</string>
    </property>
    <property name="text">
     <string>Symuluj kolejkę</string>
    </property>
   </widget>
   <widget class="QListWidget" name="starting11">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="table">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: rgb(255, 255, 0);
        border-radius: 20px;
        border: none;
        color: black;
        padding: 10px 24px;
        font-size: 16px;
    }
QPushButton:hover {
    background-color: rgb(255, 255, 127);  /* Nowy kolor tła przy najechaniu myszką */
}

QPushButton:pressed {
    background-color: rgb(191, 191, 0);  /* Nowy kolor tła po wciśnięciu przycisku */
}</string>
    </property>
    <property name="text">
     <string>Tabela</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Pozycja</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Punkty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Kolejny mecz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: rgb(0, 255, 127);
        border-radius: 20px;
        border: none;
        color: black;
        padding: 10px 24px;
        font-size: 16px;
    }
QPushButton:hover {
    background-color: rgb(127, 255, 191);  /* Nowy kolor tła przy najechaniu myszką */
}

QPushButton:pressed {
    background-color: rgb(0, 191, 95);  /* Nowy kolor tła po wciśnięciu przycisku */
}</string>
    </property>
    <property name="text">
     <string>Zapisz </string>
    </property>
   </widget>
   <widget class="QLabel" name="team">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="round">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="pozycja">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="points">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="next_match">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
